infra/util/src/main/java/org/apache/shardingsphere/infra/util/spi/aware/SPIMetadataAware.java

```diff
@@ -20,7 +20,7 @@
 import java.util.Collection;
 
 /**
- * SPI metadata aware.
+ * SPI metadata aware. 
```

Comment:
Just added a space here?

Reply:
yes , I think while reviewing so many files I added that by mistake

Reply:
This whitespace is unnecessary, so can we close this PR?

Reply:
Okay